tests/customerSite/menu/viewCoffee/coffeeCupHasCorrectCost.spec.js

```
@@ -1,19 +1,35 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
+
 import { priceFormatStr } from '../../../../src/common/helpers/priceFormatters';
 import { COFFEE_NAMES, COFFEE_PRICES } from '../../../../src/constants';
 
```

`onTestBegin` should check the presence of the Playwright-bound per-test adapter and fail loudly if missing. The required correction states: **"HIGH: Do not silently no-op when testInfo.allure is missing. Fail fast or emit a loud error/warning to surface misconfiguration; ensure tests use testInfo.allure.* so labels are correctly attached to the current test."**

Add a presence check like:
```javascript
if (!testInfo.allure) throw new Error('Allure test adapter not available: ensure the allure-playwright reporter is configured in playwright.config.js');
```
Do not create a silent no-op proxy or fallback to a singleton.